index.html

```diff
@@ -62,7 +62,67 @@
     </div>
   </header>
   <main>
-      <h1>Main items</h1>
+    <h1 class="menu-title">Menu items</h1>
+
+    <div class="container">
+      <div class="card-container">
+        <h2 class="card-price">$1200</h2>
+        <h1 class="card-title">iPhone Х</h1>
```

Comment:
Ideally, h1 should come before h2 in the document itself as well.  
Also, usually only one h1 should exist in a single page, hence, consider using `h2` and `h3` here. 😉